blocks/initialization.py

```diff
@@ -223,3 +223,23 @@ def generate(self, rng, shape):
                                                  replace=False)
             weights[i, random_indices] = values[i]
         return weights
+
+
+class Xavier(NdarrayInitialization):
+    """Initialize with Gaussian distribution with Xavier parameters.
+
+    Use the following gaussian parameters: mean=0 and var=scale/Nin
```

Comment:
I've add reference (and changed var to std so as to not confuse the reader)
however, the original paper uses a uniform distribution and I am using a gaussian....
so either I change my code or not call it Xavier or keep it as is (most  places use gaussian)